packages/slice-machine/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -206,3 +216,27 @@ export const WithBoxContent = {
   },
   parameters: { controls: { hideNoControlsWarning: true, include: [] } },
 };
+
+type OrganizationSelectProps = { disabled?: boolean };
+
+function OrganizationSelect(props: OrganizationSelectProps): ReactNode {
+  return (
+    <Select
+      {...props}
+      color="grey"
+      constrainContentWidth
+      flexContent
+      placeholder="Organization"
+      renderStartIcon={() => <GitHubIcon color={tokens.color.greyLight11} />}
```

Comment:
The icon appears further away from the text than it should. I saw you opened a thread here: [Open Slack](https://prismic-team.slack.com/archives/C0441S8SKQR/p1706279773276939)

Will we eventually change `<Select>` to use a smaller gap?